.storybook/themes/news.scss

```diff
@@ -835,6 +875,9 @@
 					padding: 0 var(--global-spacing-6) 0 0,
 					display: flex,
 				),
+				share-bar: (
+					display: flex,
```

Comment:
this is redundant right? you're using the stack here so that has display flex. it can also be removed in the mobile version. I'm not sure if I'm missing something

Reply:
@JackHowa : Yes, good catch! I implemented all styles in `default()` and set `display: none`. In `desktop()` I set `display: flex` as the top-level container is a `<Stack/>`.